Ethereum/ethereum-bayc/schema.graphql

```diff
@@ -0,0 +1,32 @@
+# To improve query performance, we strongly suggest adding indexes to any field that you plan to filter or sort by
+# Add the `@index` or `@index(unique: true)` annotation after any non-key field
+# https://academy.subquery.network/build/graphql.html#indexing-by-non-primary-key-field
+
+type Transfer @entity {
+  id: ID!
+  from: String!
+  to: String!
+  tokenId: BigInt!
```

Comment:
Should this map to a BoredApe?

Reply:
https://github.com/subquery/ethereum-subql-starter/pull/42/commits/787557af34849d9c5b0b001f6c3ff4fbb68c1319